fw/cfg.c

```diff
@@ -1230,6 +1230,44 @@ tfw_cfg_get_attr(const TfwCfgEntry *e, const char *attr_key,
 	return default_val;
 }
 
+/**
+ * Check that memory parameter is in specified range.
+ * Print an error and return non-zero if it is out of range.
+ */
+int
+tfw_cfg_mem_check_range(const char *mem, const char *mem_min,
+			const char *mem_max)
+{
+	unsigned long value = memparse(mem, NULL);
+	unsigned long min = memparse(mem_min, NULL);
+	unsigned long max = memparse(mem_max, NULL);
+
+	if (min != max && (value < min || value > max)) {
```

Comment:
Why we check that min != max ? If min == max I think we should also check value

Reply:
Fixed